chat-plugins/info.js

```diff
@@ -268,6 +91,10 @@ var commands = exports.commands = {
 			this.sendReply('|raw|' + hiddenrooms);
 		}
 	},
+	whoishelp: [
+		"/whois - Get details on yourself: alts, group, IP address, and rooms.",
+		"/whois [username] - Get details on a username: alts (Requires: % @ & ~), group, IP address (Requires: @ & ~), and rooms."
+	],
```

Comment:
I mean like this:

		whoishelp: ["/whois - Get details on yourself: alts, group, IP address, and rooms.",
			"/whois [username] - Get details on a username: alts (Requires: % @ & ~), group, IP address (Requires: @ & ~), and rooms."],